clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/TitusDeployHandler.groovy

```diff
@@ -209,6 +203,9 @@ class TitusDeployHandler implements DeployHandler<TitusDeployDescription> {
         task.updateStatus BASE_PHASE, "Finished Getting Source ASG Name Details... ${System.currentTimeMillis()}"
 
       }
+      if (!description.source.asgName) {
```

Comment:
this is effectively an `else` of the closing bracket on the line above, but...the opening bracket for that one is more than 80 lines up, so I doubt people are going to remember what it would be `else`ing.

This whole method is over 300 lines long!